### PR TITLE
Camera photo forward

### PR DESCRIPTION
**Pull request by thiagokroger**

### CODE REVIEW

**Review by katie-roberts — COMMENTED** [review]
I ran this on my Samsung S9 and the Forward button was obscured by the content at the bottom of the screen.

**Comment by thiagokroger** [comment]
> I ran this on my Samsung S9 and the Forward button was obscured by the content at the bottom of the screen.

@katie-roberts ok, I will take a look asap

**Review by katie-roberts — APPROVED** [review]
screen looks much better now.

File: src/components/views/camera/index.jsx [hunk 1]
```diff
@@ -4,14 +4,76 @@ import { Camera as ExpoCamera } from 'expo-camera'
...
+  shadow-radius: 2px;
+  shadow-opacity: 0.5;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 100px;
+  align-self: center;
+`
+
+const PhotoButtons = styled.View`
```
[the remark this answers is not in the document]
Comment (thiagokroger) [reply src/components/views/camera/index.jsx:33]: @katie-roberts you mean on the top of the screen? can you elaborate more?

**Pull request merged**